tooling/metamask/connect-to-the-network.md

```diff
@@ -27,57 +27,28 @@ Automated process of adding the Acala EVM+ network to your wallet
 ## Manual process
 
 {% hint style="info" %}
-**NOTE: The manual process example connects to the Mandala EVM+, but you can substitute the values with any of the values provided at the bottom of this page.**
+NOTE: The manual process example connects to the Mandala EVM+. For other networks can substitute the values with any of the values provided at the [network configuration](../../network/network-configuration.md) section.
 {% endhint %}
 
-In order to be able to interact with the Acala EVM+ in Mandala TC7, you first need to navigate to the **Add network** section of the MetaMask. You can find it at the bottom of the list of available networks after clicking on the currently active network.
+In order to be able to interact with the Acala EVM+ in Mandala TC8, you first need to navigate to the **Add network** section of the MetaMask. You can find it at the bottom of the list of available networks after clicking on the currently active network.
 
 ![MetaMask => Currently active network => Add network](https://1503523808-files.gitbook.io/\~/files/v0/b/gitbook-x-prod.appspot.com/o/spaces%2F-MAz4EenwXLth\_HO\_hmJ-887967055%2Fuploads%2FWz1rByJAVVr5MOdxgEaS%2FScreenshot%202022-03-02%20at%2002.22.49.png?alt=media\&token=365d2c22-49d2-4952-94cf-54a7fe154ad8)
 
-This should open up a form to add a new network to your MetaMask (you might have to unlock MetaMask before it opens). Once the form is opened, use the following information to add the Mandala TC7 network.
+This should open up a form to add a new network to your MetaMask (you might have to unlock MetaMask before it opens). Once the form is opened, use the following information to add the Mandala TC8 network.
 
-### Mandala TC7 connection details
+### Mandala TC8 connection details
 
-| **Network name**       | Mandala TC7                                              |   |
+| **Network name**       | Mandala TC8                                              |   |
 | ---------------------- | -------------------------------------------------------- | - |
 | **New RPC URL**        | `https://eth-rpc-mandala.aca-staging.network` |   |
 | **Chain ID**           | 595                                                      |   |
-| **Currency symbol**    | mACA                                                     |   |
+| **Currency symbol**    | ACA                                                     |   |
```

Comment:
this from https://github.com/ethereum-lists/chains/blob/32358c15360e23d00414af2871574ea507f7eeee/_data/chains/eip155-595.json#L8